squash/dashboard/serializers.py

```diff
@@ -0,0 +1,72 @@
+from rest_framework import serializers
+from rest_framework.reverse import reverse
+from .models import Job, Metric, Measurement
+from django.db import transaction
+
+
+class MetricSerializer(serializers.ModelSerializer):
+
+    links = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Metric
+        fields = ('metric', 'description', 'units', 'condition',
+                  'minimum', 'design', 'stretch', 'user', 'links',)
+
+    def get_links(self, obj):
+        request = self.context['request']
+        return {
+            'self': reverse('metric-detail', kwargs={'pk': obj.pk},
+                            request=request),
+         }
+
+
+class MeasurementSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Measurement
+        fields = ('metric', 'value',)
+
+    def get_links(self, obj):
+        request = self.context['request']
+        return {
+            'self': reverse('measurement-detail', kwargs={'pk': obj.pk},
+                            request=request),
+        }
+
+
+class JobSerializer(serializers.ModelSerializer):
+
+    links = serializers.SerializerMethodField()
+
+    measurements = MeasurementSerializer(many=True)
+
+    class Meta:
+        model = Job
+        fields = ('name', 'build', 'runtime',
+                  'url', 'status', 'measurements', 'links',)
+
+    """ Override the create method to create nested objects
+    from request data
+    """
+
+    def create(self, data):
+        measurements = data.pop('measurements')
```

Comment:
perhaps there is a version difference at play here?  It seems to be working for me:

```sh
$  curl -i --user sqre-admin:foo -Happlication/json" -H "Content-Type: application/json" -X GET http://localhost:8000/api/job/
HTTP/1.0 200 OK
Date: Mon, 25 Apr 2016 19:32:01 GMT
Server: WSGIServer/0.1 Python/2.7.5
Vary: Accept
X-Frame-Options: SAMEORIGIN
Content-Type: application/json
Allow: GET, POST, HEAD, OPTIONS

{"count":2,"next":null,"previous":null,"results":[{"name":"ci_cfht","build":"1","runtime":"2016-04-25T19:29:36.901849Z","url":"https://ci.lsst.codes/job/ci_cfht/1/","status":0,"measurements":[],"links":{"self":"http://localhost:8000/api/job/1/"}},{"name":"ci_cfht","build":"1","runtime":"2016-04-25T19:29:45.870848Z","url":"https://ci.lsst.codes/job/ci_cfht/1/","status":0,"measurements":[],"links":{"self":"http://localhost:8000/api/job/2/"}}]} 
$  curl -i --user sqre-admin:foo -H "Accept: application/json" -H "Content-Type: application/json" -X GET http://localhost:8000/api/metric/
HTTP/1.0 200 OK
Date: Mon, 25 Apr 2016 19:32:18 GMT
Server: WSGIServer/0.1 Python/2.7.5
Vary: Accept
X-Frame-Options: SAMEORIGIN
Content-Type: application/json
Allow: GET, POST, HEAD, OPTIONS

{"count":1,"next":null,"previous":null,"results":[{"metric":"test1","description":"Test metric insertion","units":"test","condition":"<","minimum":8.0,"design":5.0,"stretch":3.0,"user":5.0,"links":{"self":"http://localhost:8000/api/metric/test1/"}}]}
```